test/src/main/java/org/axonframework/test/saga/FixtureConfiguration.java

```diff
@@ -211,6 +212,22 @@ FixtureConfiguration registerDeadlineHandlerInterceptor(
     FixtureConfiguration registerListenerInvocationErrorHandler(
             ListenerInvocationErrorHandler listenerInvocationErrorHandler);
 
+    /**
+     * Registers a {@link ResourceInjector} within this fixture. This approach can be used if a custom {@code
+     * ResourceInjector} has been built for a project which the user wants to take into account when testing it's
+     * sagas.
+     * <p>
+     * The provided {@code resourceInjector} will be paired with the fixture's default {@code ResourceInjector} to keep
+     * supporting the {@link #registerResource(Object)} and {@link #withTransienceCheckDisabled()} methods. Note that
+     * <b>first</b> the default injector is called, and after that the given {@code resourceInjector}. This approach
+     * ensures the fixture's correct workings for default provided resources, like the {@link EventBus} and {@link
```

Comment:
Not sure I understand this. If the custom injector comes second, doesn't that give it the opportunity to override certain dependencies? If we want to force our own, the default injector should come last.

Reply:
My train of thought was to keep support for the `registerResource(Object)` and `withTransienceCheckDisabled()` methods workable, but allow the users to override the entirety of resource injections or to append to it. 

To achieve both complete override and appending, I felt the custom implementation should come second.

Reply:
Discussed offline that the current approach is fair, but that a more direct caution sign should be given in the javadoc to describe what happens if a custom all overriding `ResourceInjector` is provided.